src/citrine/informatics/predictor_evaluator.py

```diff
@@ -23,28 +24,24 @@ def get_type(cls, data) -> Type[Serializable]:
             "HoldoutSetEvaluator": HoldoutSetEvaluator
         }[data["type"]]
 
-    def _attrs(self) -> List[str]:
-        raise NotImplementedError  # pragma: no cover
-
     def __eq__(self, other):
-        try:
-            return all([
-                self.__getattribute__(key) == other.__getattribute__(key) for key in self._attrs()
-            ])
-        except AttributeError:
+        if isinstance(other, Serializable):
+            return self.dump() == other.dump()
+        else:
             return False
 
     @property
+    @abstractmethod
     def responses(self) -> Set[str]:
         """Responses to compute metrics for."""
-        raise NotImplementedError  # pragma: no cover
```

Comment:
Functionally equivalent, minus a change in exception type.  If that feels breaking, then we can do a minor bump.

Reply:
According to the docs, `@abstractmethod` doesn't mean anything unless the class's metaclass is `ABCMeta`. I don't think that's the case here, so that decorator means nothing?

If the intent is for the metaclass to be `ABCMeta`, then I disagree they're functionally equivalent, as that would prevent even instantiating an instance of the class, not only impacting the method.

Reply:
Removed.